src/clarity-angular/button/_toggles.clarity.scss

```diff
@@ -58,7 +58,7 @@ $toggle-transition-animation: ease-in;
             height: $toggle-switch-base-height;
             width: $toggle-switch-base-width;
             border: 2px solid;
```

Comment:
Do we want to convert this to rem?

Reply:
No. I tested this with a variety of sizes and it held (surprisingly). It needs this sizing here due to how particular the toggle is about positioning, etc.